coderedcms/views.py

```diff
@@ -125,3 +135,120 @@ def robots(request):
         {'robots': robots},
         content_type='text/plain'
     )
+
+@transaction.atomic()
+def import_pages(import_data, parent_page):
```

Comment:
For all of these utility functions and class, would it make more sense to put those in a separate file, maybe `importexport.py` or `utils.py` or something? So as not to clutter the actual views. I would picture only the actual view function(request) being here.